typedb-src/modules/ROOT/pages/admin/update.adoc

```diff
@@ -40,13 +40,13 @@ information on TypeDB server database compatibility see the table below.
 | Database encoding Version | Compatible TypeDB Versions
 
 | 0
-| 2.0.0 - 2.5.0
+| 2.0.0 to 2.5.0
 
 | 1
-| 2.6.0 - 2.7.1
+| 2.6.0 to 2.7.1
 
 | 2
-| 2.8.0 - latest
+| 2.8.0 to latest
```

Comment:
I think this is the only use of "latest."

Reply:
Yes, but this table rarely gets updated. It's unique, as it's not about the connection but the storage. 
Do you think we can leave the latest here?

Reply:
Again, up to you, but I noticed it.

Reply:
FYI this is why i preferred `-` because you can leave it open `2.8.0 - ` implies open ended